policyengine_us/parameters/gov/states/mn/tax/income/subtractions/sources.yaml

```diff
@@ -1,4 +1,4 @@
-description: Minnesota subtracts these variables from US AGI when computing its taxable income.
+description: Minnesota subtracts these variables from federal adjusted gross income when computing its taxable income.
```

Comment:
just confirming this is when computing taxable income not state agi?

Reply:
Yes, when computing MN taxable income